part-2.py

```diff
@@ -4,13 +4,41 @@
 # the appropriate comment.
 
 # search
-
+def search(array, query):
```

Comment:
👍 just note that `array[:-1]` also copies the array (except the last element) so this ends up being O(n^2) for time/space